webapp/src/main/scala/org/allenai/common/webapp/InfoRoute.scala

```diff
@@ -37,14 +39,14 @@ class InfoRoute(val info: Map[String, String] = Map.empty) {
       pathEndOrSingleSlash {
         respondWithMediaType(MediaTypes.`application/json`) {
           complete {
-            info.toJson.prettyPrint
+            infoWithTime.toJson.prettyPrint
```

Comment:
It's a little odd that this copy is used in place of `info` in some places, but not everywhere.

I'd prefer if this were added in the same pattern as `withName` and `withVersion` - like `withStartupTime` (either taking the startup time value to use, or else no-args & capturing the current time).